src/client/helpers/analytics.js

```diff
@@ -0,0 +1,12 @@
+import ReactGA from 'react-ga';
+
+const GA_TRACKING_ID = 'UA-XXXXX-Y';
```

Comment:
This variable should be placed in [config](https://github.com/paralect/nextjs-landing-starter/tree/4958fc12e4f903ddf2aae92235c6c12271693ba1/src/server/config/environment)
[Here](https://github.com/paralect/nextjs-landing-starter/blob/4958fc12e4f903ddf2aae92235c6c12271693ba1/src/client/helpers/api/api.client.js#L2) is example of how to take it from config

Reply:
@SobakaSlava Where exactly in the config?

Reply:
Near the `apiUrl` variable